site/specs-temp/numbers.json

```
@@ -23336,7 +23369,7 @@
             "type": "object",
             "properties": {
               "OrderHistory": {
-                "type": "array",
+                "type": "object",
                 "items": {
                   "$ref": "#/components/schemas/OrderHistory"
                 }
```

I dont think this is valid openapi - if it is of type `object` it should have properties or be a $ref - an object should not have `items` IIRC

```suggestion
                "type": "object",
                "$ref": "#/components/schemas/OrderHistory"
```